usaspending_api/references/management/commands/fix_zip_codes.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+class Command(BaseCommand):
```

Comment:
I believe this should be fine as a stand alone command since it doesn't have to be part of the nightly load process and just needs to be run once.